benchmark/harness_results.go

```diff
@@ -43,33 +48,33 @@ func (r *BenchResult) EvergreenPerfFormat() ([]interface{}, error) {
 
 	out := []interface{}{
 		map[string]interface{}{
-			"name": r.Name + "-throughput",
-			"results": map[string]interface{}{
-				"1": map[string]interface{}{
-					"seconds":        r.Duration.Round(time.Millisecond).Seconds(),
-					"ops_per_second": r.getThroughput(median),
-					"ops_per_second_values": []float64{
-						r.getThroughput(min),
-						r.getThroughput(max),
-					},
+			"info": map[string]interface{}{
+				"test_name": r.Name + "-throughput",
+				"args": map[string]interface{}{
```

Comment:
The instructions in the [migration description](https://github.com/10gen/performance-tooling-docs/blob/main/getting_started/performance_monitoring_setup.md#legacy-data-migration) list a "compression" argument in this subsection, as well. I'm not clear what that argument means or whether we need to encode it.

Reply:
My understanding is that `args` can be an arbitrary set of values to describe the test scenario. I think "compression" was meant as an example.